src/server/list_family.cc

```diff
@@ -625,10 +672,7 @@ OpResult<uint32_t> OpRem(const OpArgs& op_args, string_view key, string_view ele
 
     auto it = ql->GetIterator(where);
     auto is_match = [&](const QList::Entry& entry) {
-      if (is_int) {
-        return entry.is_int() && entry.ival() == ival;
-      }
-      return entry == elem;
+      return (is_int && entry.is_int() && entry.ival() == ival) || (entry == elem);
```

Comment:
First time I've written it incorrectly and then updated the comment
is_int ? entry.is_int() && entry.ival() == ival : entry == elem;